### ISSUE TITLE
 delete company not clear

### ISSUE THREAD

**Issue by mahmoud-abdalla** [moar55/se-project-testing#48]
1: Severity (medium)

2: Reported by
Mahmoud Yasser

3: Description
deleting a company is not clear as you have to press the big red button and scroll down to be able to view the companies to delete them. and the user is not direct to do so.

4: Steps to reproduce issue: 
1) login as admin
2)press on the big red button 
3)scroll down to view the companies

5: Expected result
label over the red button so that the admin can know that it deletes companies. and the be directed to scroll down to view the companies

**Comment by safya-alzayat** [comment]
can you please mention the steps that lead to this as when i tried i can both delete and reject service providers

**Comment by mahmoud-abdalla** [comment]
wala tez3aly, there are no steps. can you delete a company you no longer want?

**Comment by safya-alzayat** [comment]
yes by clicking on the large X in the admin homepage and scrolling down there will be 3 tabs click on the service provider tab and scroll down there will be a list of all service providers click on the one you want to delete
